website/www/site/content/en/powered-by/commercial/eBay.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Feature Powered by Apache Beam - Beyond Lambda"
+type: commercial
+categories:
+  - Online & Traditional Retail
+icon: /images/logos/powered-by/eBay.png
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Developed at Spotify and built on top of Apache Beam for Python, Klio is an open source framework that lets researchers and engineers build smarter data pipelines for processing audio and other media files, easily and at scale.
```

Comment:
This is the copy for Klio, not eBay

Reply:
Yeah sorry about this, I pushed the right one

Reply:
LG, thanks